src/components/ui/FloatingItems.jsx

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
 
-const FloatingItems = ({ items }) =>
-  <div className="floatingItems">
+const FloatingItems = ({ items, position }) =>
+  <div
+    className="floatingItems"
+    style={position === 'left'
```

Comment:
Why not manage this 12 as padding, as this seems to be what it is I think.
And I guess the transform is about putting things above the scale and i button? I guess it would be better visually to hide them (or lift them above the items) when `position === 'left'`.

Reply:
Thanks, I added a commit to hide the scale element on the map while the button is displayed, on mobile.